sklearn/ensemble/_bagging.py

```diff
@@ -538,6 +564,33 @@ def estimators_samples_(self):
     def n_features_(self):
         return self.n_features_in_
 
+    def get_metadata_routing(self):
+        """Get metadata routing of this object.
+
+        Please check :ref:`User Guide <metadata_routing>` on how the routing
+        mechanism works.
+
+        Returns
+        -------
+        routing : MetadataRouter
+            A :class:`~utils.metadata_routing.MetadataRouter` encapsulating
+            routing information.
+        """
+        base_estimator = self._validate_estimator(self._get_estimator())
+        router = (
+            MetadataRouter(owner=self.__class__.__name__)
+            # no add_self(self) because the bagging metaestimator does not use
+            # fit_params
+            .add(
+                base_estimator=base_estimator,
+                method_mapping=MethodMapping().add(callee="fit", caller="fit"),
+            )
```

Comment:
It doesn't seem to be necessary, when using the deprecated `base_estimator`, we get a warning for it but it's not necessary to separately add a separate mapping for it because the correct estimator is resolved beforehand. There is a test for it (`test_base_estimator_argument_missing_fit_request_warns`).